packages/core/test/utilities.ts

```diff
@@ -0,0 +1,8 @@
+export const generateSeed = (length = 81) => {
```

Comment:
Is it able to `generateHash` and produce any length we pass? Like maybe I want to get a tag that is 27 long tryte.

Reply:
`generateHash` looks like an underlying function, and `generateSeed` should use it.
By the way, is it useful to provide the `generateSeed()` to the user of the `@tangleid/core`? 🤔  

The code here is only provided for testing.

Reply:
I see. If this is only for testing I think it could be a fixed one.